File: FrogJmp/php/TestFrogJmp.php
<?php

require_once 'FrogJmp.php';

class Test extends PHPUnit_Framework_TestCase {

	protected $frog_jmp;

	protected function setup() {
		$this->frog_jmp = new \Codility\FrogJmp;
	}

	public function testSolution() {
		$this->assertEquals(3, $this->frog_jmp->solution(10,85,30));
	}

}

File: FrogJmp/ruby/TestFrogJmp.rb
require_relative 'FrogJmp'
require 'minitest/autorun'

class Test < MiniTest::Test

	def setup
		@frog_jmp = FrogJmp.new
	end

	def test_solution
		assert_equal 3, @frog_jmp.solution(10,85,30)
	end

end
